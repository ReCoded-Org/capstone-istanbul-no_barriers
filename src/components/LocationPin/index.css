.pin-icon {
    height: 3rem;
    width: auto;
}