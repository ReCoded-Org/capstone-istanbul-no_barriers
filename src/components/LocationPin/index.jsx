import React from "react";
import "./index.css";

const LocationPin = ({ address }) => (
  <div>
    <span className="iconify" data-icon="mdi:map-marker" data-inline="false" />
    <p>{address}</p>
  </div>
);

export default LocationPin;
