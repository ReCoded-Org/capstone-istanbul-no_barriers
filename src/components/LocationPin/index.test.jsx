import React from "react";
import renderer from "react-test-renderer";
import LocationPin from "./index";

it("LocationPin renders correctly", () => {
  const tree = renderer.create(<LocationPin address="Ankara" />).toJSON();
  expect(tree).toMatchSnapshot();
});
