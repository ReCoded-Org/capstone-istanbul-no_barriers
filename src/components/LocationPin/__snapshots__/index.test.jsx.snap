// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationPin renders correctly 1`] = `
<div>
  <span
    className="iconify"
    data-icon="mdi:map-marker"
    data-inline="false"
  />
  <p>
    Ankara
  </p>
</div>
`;
